account-kit/core/src/actions/disconnect.ts

```diff
@@ -46,17 +46,36 @@ export async function disconnect(config: AlchemyAccountsConfig): Promise<void> {
 // Persistence of Wallet Connect connection state on error.
 const clearWalletConnectStore = () => {
   // Open Wallet Connect Indexed DB
+  let walletConnectDBExists = true;
   const dbOpenRequest = indexedDB.open("WALLET_CONNECT_V2_INDEXED_DB");
```

Comment:
do we need to do any other checks here? does `indexDb.open` ever throw?

Reply:
Yeah, but we're already handling the error case for that with the `dbOpenRequest.onerror()` handler below